providers/aws/dynamodb/tables.go

```diff
@@ -47,14 +84,32 @@ func Tables(ctx context.Context, client ProviderClient) ([]Resource, error) {
 			}
 		}
 
+		tableDetails, err := dynamodbClient.DescribeTable(ctx, &dynamodb.DescribeTableInput{
+			TableName: aws.String(table),
+		})
+
+		if err != nil {
+			log.Errorf("ERROR: Failed to query DynamoDB table details: %v", err)
+		}
+
+		if tableDetails != nil && tableDetails.Table != nil && tableDetails.Table.ProvisionedThroughput != nil {
```

Comment:
```suggestion
		if tableDetails.Table != nil && tableDetails.Table.ProvisionedThroughput != nil {
```
`tableDetails` won't be nil if there's no error